### PR TITLE
add support for reading snyk json from stdin

### LINKED ISSUE #5: Feat: enable the CLI to receive Snyk's JSON input from stdin
## Description

In essence, `pie-my-vulns` uses the Snyk CLI behind the scenes to scan and use the JSON output of that to chart the vulnerabilities. If someone has the Snyk CLI already installed, they could do something like `snyk test --json | npx pie-my-vulns`. To enable that we should support getting the JSON input from stdin and also when that takes place there's no need to run the audit within `pie-my-vulns`'s own code.

### PR DESCRIPTION
**Pull request by DrFaust92**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

Closes #5

## Motivation and Context

This Change allows passing a Snyk JSON to the cli and save the call to auth

## Checklist:

- [ ] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

File: __tests__/__e2e__/cli.e2e.js [hunk 1]
```diff
@@ -65,4 +66,15 @@ describe('End-to-End CLI', () => {
     expect(err).toBe(undefined)
     expect(stdout).toContain('0 vulnerabilities found')
   })
+
+  test('CLI should be able to read Snyk Json from stdin', async () => {
```
Comment (lirantal) [inline __tests__/__e2e__/cli.e2e.js]:
```suggestion
  test('CLI should be able to read Snyk JSON from stdin', async () => {
```

**Comment by lirantal** [comment]
This is very close but not exactly what I had in mind. I was referring to the CLI being able to accept the JSON input as Standard Input (STDIN), such as: `cat snyk.json | pie-my-vulns`, and not as a command line argument.

There are minimal changes required really for the code itself and the test probably needs to be adjusted to spawn a shell that can do pipes to simulate this condition.

**Comment by lirantal** [comment]
@DrFaust92 if you wanted to take some of my input on this to still get the PR landed

**Comment by DrFaust92** [comment]
Hey, sorry didn't have time to get around to addressing your comments. I'll be able to take a look on the upcoming weekend.

**Comment by lirantal** [comment]
Sure thing. Stay safe and healthy ❤️

**Comment by lirantal** [comment]
closing in favor of #72

**Pull request closed without merging**